controllers/storagecluster/noobaa_system_reconciler_test.go

```diff
@@ -304,11 +315,16 @@ func TestSetNooBaaDesiredState(t *testing.T) {
 		assert.NotEmptyf(t, noobaa.Labels, "[%s] expected noobaa Labels not found", c.label)
 		assert.Equalf(t, noobaa.Labels["app"], "noobaa", "[%s] expected noobaa Label mismatch", c.label)
 		assert.Equalf(t, noobaa.Name, "noobaa", "[%s] noobaa name not set correctly", c.label)
-		assert.Equal(t, *noobaa.Spec.DBStorageClass, fmt.Sprintf("%s-ceph-rbd", c.sc.Name))
+		if c.label == "case 6" {
+			assert.Equal(t, *noobaa.Spec.DBStorageClass, noobaDbDefaultStorageClass)
+			assert.Equal(t, *noobaa.Spec.PVPoolDefaultStorageClass, noobaDbDefaultStorageClass)
+		} else {
+			assert.Equal(t, *noobaa.Spec.DBStorageClass, fmt.Sprintf("%s-ceph-rbd", c.sc.Name))
+			assert.Equal(t, *noobaa.Spec.PVPoolDefaultStorageClass, fmt.Sprintf("%s-ceph-rbd", c.sc.Name))
+		}
```

Comment:
Rather than doing this, extend the cases struct to have a new "dbStorageClassName" string field, or something, and set it to `generateNameForCephBlockPoolSC(sc, "")` as the default if not specified. Alternatively you could just do an `if` to check if `Spec.MultiCloudGateway.DbStorageClassName` (watch out for null pointers!) exists and use that if so. In the end I just don't want to see use using two Equal() calls on the same field with different values depending on use case label.